Client/Teztez/Resources/Suporting Files/Assets.xcassets/ActivitiesIntro/backwards-close-icon.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "backwards-close-icon.pdf"
+    }
+  ],
```

Comment:
There are some caveats when you add vector (`pdf`) assets to XCode project. 
[Check it here](https://www.hackingwithswift.com/example-code/xcode/how-to-use-vector-images-in-your-asset-catalog)  (Basically it's just choosing **Single Scale** option and ticking **Preserve Vector Data** checkbox)